README.md

```diff
@@ -40,19 +40,19 @@ development branch when they think their feature or bug fix is ready.
 
 The patch will be accepted if there is broad consensus that it is a
 good thing.  Developers should expect to rework and resubmit patches
-if they don't match the project's coding conventions (see coding.txt)
+if they don't match the project's coding conventions (see [coding.txt](doc/coding.txt))
 or are controversial.
 
 The master branch is regularly built and tested, but is not guaranteed
-to be completely stable. Tags are regularly created to indicate new
+to be completely stable. [Tags](https://github.com/gridcoin/Gridcoin-Research/tags) are regularly created to indicate new
 stable release versions of Gridcoin.
 
 Feature branches are created when there are major new features being
 worked on by several people.
 
-From time to time a pull request will become outdated. If this occurs, and
-the pull is no longer automatically mergeable; a comment on the pull will
-be used to issue a warning of closure. The pull will be closed 15 days
+After a set period of time, an open pull request will become outdated. If this occurs, and
```

Comment:
I think we should remove this inherited section. PRs do not need these strict rules.